clientUtils/owidTypes.ts

```diff
@@ -265,6 +265,14 @@ export interface FullPost {
     glossary: boolean
 }
 
+export interface Detail {
```

Comment:
This interface has been defined in other branches already. Once everything's merged I'll prune down the duplication.